semeio/workflows/localisation/local_script_lib.py

```diff
@@ -744,6 +777,155 @@ def add_ministeps(
         ert_local_config.getUpdatestep().attachMinistep(ministep)
 
 
+def get_corr_group_spec(correlations_spec_list, name):
```

Comment:
No test for this function

Reply:
Is only used in the test_integration.py functions.